app/views/layouts/simulator.html.erb

```diff
@@ -3,7 +3,11 @@
   <head>
     <title>CircuitVerse - Digital Circuit Simulator online</title>
     <%= csrf_meta_tags %>
+    <!-- bring locale value from backend to frontend -->
+    <script>window.locale = "<%= I18n.locale %>"</script>
     <script src="/js/pace.js"></script>
+    <!--load banana-i18n library-->
+   <script src="/js/banana-i18n.js"></script>
```

Comment:
Do you still need this as we are fetching from npm?

Reply:
I did mistake here, now importing from node_modules. corrected in new commit!